tests/one_to_one/models.py

```diff
@@ -118,3 +118,18 @@ class Director(models.Model):
     is_temp = models.BooleanField(default=False)
     school = models.OneToOneField(School)
     objects = DirectorManager()
+
+
+class UnsavedOneToOneField(models.OneToOneField):
+    # A OneToOneField which can point to an unsaved object
+    unsaved_instances_allowed = True
```

Comment:
Could you add documentation for this? I'd suggest the name `allow_unsaved_instance_assignment`